model/config.go

```diff
@@ -3347,8 +3347,6 @@ func (o *Config) Sanitize() {
 	}
 
 	*o.SqlSettings.DataSource = FAKE_SETTING
-	o.SqlSettings.DataSourceReplicas = []string{FAKE_SETTING}
```

Comment:
The changes in this file are reverting the previous approach of sanitization/desanitization.